src/spineml/standalone_simulator/main.cc

```diff
@@ -0,0 +1,55 @@
+// SpineML simulator includes
+#include "simulator.h"
+
+using namespace SpineMLSimulator;
+
+int main(int argc, char *argv[])
+{
+    try
+    {
+        if(argc < 2) {
+            throw std::runtime_error("Expected experiment XML file passed as arguments");
+        }
+
+        // Read min log severity from command line
+        const plog::Severity minSeverity = (argc > 3) ? (plog::Severity)std::stoi(argv[3]) : plog::info;
```

Comment:
Should this be ``argc > 2`` and ``argv[2]``? Or are you using the second command line argument for something else?

Reply:
Yeah - the 2nd command line argument is working directory (the default used for SpineCreator is very irritating)